ush/rocoto/setup_workflow_fcstonly.py

```diff
@@ -132,7 +132,7 @@ def get_definitions(base):
     strings.append('\t<!ENTITY SCHEDULER  "%s">\n' % scheduler)
     strings.append('\n')
     strings.append('\t<!-- Toggle HPSS archiving -->\n')
-    strings.append('\t<!ENTITY ARCHIVE_TO_HPSS "YES">\n')
+    strings.append('\t<!ENTITY ARCHIVE_TO_HPSS "%s">\n' % base['HPSSARCH'])
```

Comment:
String modulo has long been discouraged in favor of using format of f-strings. We should start using f-strings whenever possible, as it is the easiest and most readable, thought it doesn't have to be right now.

Reply:
Gotcha. Am happy to redo things to f-strings when we go through to do mass updating like we've discussed, I kept things consistent with how the script was doing this for now.